.github/workflows/cypress.yml

```diff
@@ -0,0 +1,36 @@
+name: E2E Tests
+
+on: [push]
```

Comment:
also, why not doing it on pull requests as well ?

Reply:
Yes we probably should !

Reply:
Ok, let's do it then 🙂